src/components/avatars.css

```diff
@@ -0,0 +1,39 @@
+.avatar {
+  width: 80px;
```

Comment:
Would prefer to use `rem` for large units,
or also use CSS variables as well (pre-defined sizes)

Reply:
yes will add some variable for sizes